idofront-util/src/main/kotlin/com/mineinabyss/idofront/plugin/Plugins.kt

```diff
@@ -22,7 +22,10 @@ object Plugins {
         getOrNull<T>()?.let { isEnabled(it) } ?: false
 
     /** Checks if a [plugin] exists and is enabled. */
-    fun isEnabled(plugin: Plugin) = Bukkit.getPluginManager().isPluginEnabled(plugin)
+    fun isEnabled(plugin: Plugin) = runCatching { Bukkit.getPluginManager().isPluginEnabled(plugin) }.isSuccess
```

Comment:
⚠️ I don't think this works if the plugin is loaded into the jar but disabled (afterwards or maybe disables itself), since isPluginEnabled will return false, but runCatching will finish successfully. Maybe
`runCatching.getOrElse { false }` or whatever the syntax is?

Reply:
fixed 👍